zuul.d/projects.yaml

```diff
@@ -742,6 +742,7 @@
     default-branch: main
     templates:
       - system-required
+      - ansible-navigator-jobs
```

Comment:
you should add these into ansible-zuul-jobs as a project-template

Reply:
ah, ignore what I said. This is missing a depends-on

Reply:
The PR for ansible-zuul-jobs is on the way to 'merge'. I will recheck this PR after the merge.